customer.go

```diff
@@ -99,27 +99,38 @@ type Customers struct {
 // are not returned by the Dwolla API. As such, we don't want fields to be
 // unset during marshaling.
 type CustomerRequest struct {
-	FirstName              string         `json:"firstName,omitempty"`
-	LastName               string         `json:"lastName,omitempty"`
-	Email                  string         `json:"email,omitempty"`
-	IPAddress              string         `json:"ipAddress,omitempty"`
-	Type                   CustomerType   `json:"type,omitempty"`
-	Status                 CustomerStatus `json:"status,omitempty"`
-	DateOfBirth            string         `json:"dateOfBirth,omitempty"`
-	SSN                    string         `json:"ssn,omitempty"`
-	Phone                  string         `json:"phone,omitempty"`
-	Address1               string         `json:"address1,omitempty"`
-	Address2               string         `json:"address2,omitempty"`
-	City                   string         `json:"city,omitempty"`
-	State                  string         `json:"state,omitempty"`
-	PostalCode             string         `json:"postalCode,omitempty"`
-	BusinessClassification string         `json:"businessClassification,omitempty"`
-	BusinessType           string         `json:"businessType,omitempty"`
-	BusinessName           string         `json:"businessName,omitempty"`
-	DoingBusinessAs        string         `json:"doingBusinessAs,omitempty"`
-	EIN                    string         `json:"ein,omitempty"`
-	Website                string         `json:"website,omitempty"`
-	Controller             Controller     `json:"controller,omitempty"`
+	FirstName              string             `json:"firstName,omitempty"`
```

Comment:
@darland Is this just whitespace changes? I'm using `go fmt`.

Reply:
@kolanos Not really, I have added a pointer to Controller and changed nested struct from Controller to new ControllerRequest (you can see it below).